src/main/java/hudson/plugins/sonar/SonarRunnerBuilder.java

```diff
@@ -284,6 +285,13 @@ private void handleErrors(Run<?, ?> build, TaskListener listener, SonarRunnerIns
       errorMessage += Messages.SonarRunner_GlobalConfigNeeded();
     }
     e.printStackTrace(listener.fatalError(errorMessage));
+
+    try {
+      SonarUtils.addUrlActionTo(build);
+    } catch (IOException e1) {
+      e1.printStackTrace(listener.fatalError(errorMessage));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.internal.sonarsource.com/coding_rules#rule_key=squid%3AS1148)